implementations/rust/okta/management_app/src/main.rs

```diff
@@ -0,0 +1,204 @@
+#[macro_use]
+extern crate lazy_static;
+
+mod config;
+
+use ockam::{
+    vault::*,
+    vault::types::*,
+    vault_software::*,
+};
+use oktaplugin::{Messages, OckamMessages, OckamRole, KeyEstablishment};
+use rand::prelude::*;
+use std::{
+    fs,
+    io::{self, Write},
+    net::TcpStream,
+    path::Path,
+    sync::Mutex,
+};
+use structopt::StructOpt;
+use oktaplugin::Messages::OktaRequest;
+use std::collections::BTreeMap;
+use std::borrow::BorrowMut;
+use std::thread::sleep;
+
+const FILE_NAME: &str = ".env";
+
+lazy_static! {
+    static ref VAULT: Mutex<DefaultVault> = Default::default();
+}
+
+fn main() {
+    let cfg =
+        if Path::new(FILE_NAME).is_file() {
+            let contents = fs::read_to_string(FILE_NAME).unwrap();
```

Comment:
Would be nice to remove unwraps

Reply:
CLI's are okay to have unwraps. This is just a demo anyway